packages/react-card/src/components/Card/useCardStyles.ts

```diff
@@ -6,21 +6,50 @@ import type { CardState } from './Card.types';
  */
 const useStyles = makeStyles({
   root: theme => ({
-    // TODO Add default styles for the root element
+    display: 'flex',
+    flexDirection: 'column',
+    overflow: 'hidden',
+
+    boxShadow: theme.alias.shadow.shadow4,
+    color: theme.alias.color.neutral.neutralForeground1,
+    backgroundColor: theme.alias.color.neutral.neutralBackground1,
+
+    // Size: medium
+    // TODO: Validate if we should use a token instead + the unit of said token
+    // TODO: Explore alternate way of applying padding
+    padding: '12px',
+    gap: '12px',
+    borderRadius: theme.global.borderRadius.medium,
   }),
 
-  // TODO add additional classes for different states and/or slots
+  interactive: theme => ({
+    cursor: 'pointer',
+    ':hover': {
+      backgroundColor: theme.alias.color.neutral.neutralBackground1Hover,
+    },
+    ':active': {
+      backgroundColor: theme.alias.color.neutral.neutralBackground1Pressed,
+    },
+  }),
 });
 
 /**
  * Apply styling to the Card slots based on the state
  */
 export const useCardStyles = (state: CardState): CardState => {
   const styles = useStyles();
-  state.className = mergeClasses(styles.root, state.className);
-
-  // TODO Add class names to slots, for example:
-  // state.mySlot.className = mergeClasses(styles.mySlot, state.mySlot.className);
+  state.className = mergeClasses(
+    styles.root,
+    (state.onClick ||
+      state.onMouseUp ||
+      state.onMouseDown ||
+      state.onPointerUp ||
+      state.onPointerDown ||
+      state.onTouchStart ||
+      state.onTouchEnd) &&
```

Comment:
To be extracted into a separate utility once we define our standard of interactivity.

Reply:
I would avoid making styling assumptions based on event listeners used as it might lead to higher amount of custom overrides. Having explicit prop like `interactive?: boolean` is more robust. You can consider changing the default value of it based on the listeners (like if onClick is used and interactive is not, it would still be interactive, unless consumer specifies `interactive={false}`)

Reply:
I don't see what the point of an `interactive` prop would be if the styling is still applied based on events. To me this is a visual accessibility feature that we want to be there.
If the user wants actions with no visual feedback or vice-versa, this seems like behavior we wouldn't want to encourage.

Reply:
That's a valid point and even more of a reason to have this logic abstracted into a separate utility which can reason about it internally based on `state`.

I am, however, a bit afraid of a prop like `interactive?: boolean` as it exposes styling internals. This will also mean we run into a situation where we decouple styling interactivity and functional interactivity.
- What should happen if the prop is true but there are no handlers?
  - Should it look interactive but when users try to manipulate it, it does nothing? 
- What should happen if the prop is false and there are handlers?
  - Should it disable those handlers?
  - Should it look non-interactive? But how are we then handling cases like keyboard focus?
  
If we believe that current component spec will lead to high amount of custom overrides then in my opinion the component design is not done properly and interactivity should be better defined within the spec. Otherwise, this would just be a premature optimisation for a problem that might not be a problem at all, especially when taking into consideration there already is a mechanism for users to override the styling if they want to opt-into less common scenarios.

Reply:
Looking at the figma design specs, there is no requirement that these interactive styles are linked to any kind of event listeners.

Why does interactive necessarily mean a mouse click ? why can a mouseenter not define interactive behaviour ? If a user wants to have hover styles, they would currently need to add an empty click listener to do that.

The event listeners also miss out on `contextmenu` event, should that also define interactivity ?

Adding an interactive prop means that we allow maximum flexibility for users since the designers who will work with these components can be from different product groups and might not associate things like hover styles with  a click.

Reply:
@andrefcdias @jurokapsiar @ling1726 I opened an issue which covers this. I feel this discussion targets more than just the card and feel that extra visibility of a separate issue might be useful. If you could raise your points there it would be greatly welcome. 🙏 

https://github.com/microsoft/fluentui/issues/19664

Reply:
This is why we need to define interactivity. My personal opinion is that if something has an action, it's interactive and needs visual feedback of being actionable. A mouseenter event is not an event that, to me, would define interactivity.
End of the day, this is something that should be opt out. If the user doesn't define anything, we provide them with a good baseline.

My suggestion is that we go with the current solution, keeping in mind that this is definitely not set in stone. There's already the pretext that breaking changes will happen in the next phases, so let's not dwell on this right now and migrate this discussion into the spec PR to define the definite solution for it, unless this is of high importance to have defined right now.
What do you think?